app/views/catalog/_thumbnail.html.erb

```diff
@@ -1,4 +1,7 @@
-<% if presenter(document).thumbnail.exists? && tn = presenter(document).thumbnail.thumbnail_tag({ alt: '' }, 'aria-hidden': true, tabindex: -1, counter: document_counter_with_offset(document_counter)) %>
+<% if presenter.thumbnail.exists? &&
```

Comment:
For backwards compatibility, maybe we should provide the old behavior if the presenter local isn't available?

Reply:
Good catch. I've added https://github.com/projectblacklight/blacklight/pull/2304/files#diff-1228e931859595c68c525cca60385f73R1-R3 to take care of this.